src/pages/tiers.js

```diff
@@ -0,0 +1,110 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import withData from '../lib/withData';
+import withIntl from '../lib/withIntl';
+import Header from '../components/Header';
+import Body from '../components/Body';
+import Container from '../components/Container';
+import TierCard from '../components/TierCard';
+import Footer from '../components/Footer';
+import Link from '../components/Link';
+import Logo from '../components/Logo';
+import Loading from '../components/Loading';
+import ErrorPage from '../components/ErrorPage';
+import { withUser } from '../components/UserProvider';
+import { addCollectiveData } from '../graphql/queries';
+import { P, H3 } from '../components/Text';
+
+class TiersPage extends React.Component {
+  static getInitialProps({ query: { collectiveSlug } }) {
+    return { slug: collectiveSlug };
+  }
+
+  static propTypes = {
+    slug: PropTypes.string, // from getInitialProps, for addCollectiveData
+    query: PropTypes.object, // from getInitialProps
+    data: PropTypes.object.isRequired, // from withData
+    LoggedInUser: PropTypes.object,
+    intl: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { LoggedInUser, data = {}, intl } = this.props;
+
+    if (!data || !data.Collective) {
+      return <ErrorPage data={data} />;
+    }
+
+    const collective = data.Collective;
+    const logo = collective.image || get(collective.parentCollective, 'image');
+
+    return (
+      <div>
+        <Header
+          title={'Tiers'}
+          description="All the collectives that you are giving money to"
```

Comment:
This is not a correct description for this page

Reply:
What do you think the title should be?